osimpipeline/task.py

```diff
@@ -96,29 +105,183 @@ def __init__(self, trial):
         super(TrialTask, self).__init__(trial.subject)
         self.trial = trial
 
+class SetupTask(TrialTask):
+    def __init__(self, tool, trial, 
+                 model_to_adjust=None, 
+                 adjusted_model=None,
+                 gait_cycles='concatenated'):
+        super(SetupTask, self).__init__(trial)
+        self.tool = tool
+        self.trial = trial
+        self.name = '%s_%s_setup' % (trial.id, self.tool)
+        self.tool_path = os.path.join(trial.results_exp_path, self.tool)
+        self.doit_path = self.study.config['doit_path']
+        self.gait_cycles=gait_cycles
+        self.source_extloads_fpath = os.path.join(self.trial.rel_path, 
+                self.tool, 'external_loads.xml')
+        self.results_extloads_fpath = os.path.join(self.tool_path,
+                os.path.basename(self.source_extloads_fpath))
+        self.source_tasks_fpath = os.path.join(self.trial.rel_path, self.tool,
+                'tasks.xml')
+        self.results_tasks_fpath = os.path.join(self.tool_path, 
+                os.path.basename(self.source_tasks_fpath))
+
+        # Specify model to be adjusted by RRA
+        if not model_to_adjust:
+             self.model_to_adjust = '%s.osim' % self.subject.name
+        else:
+            self.model_to_adjust = model_to_adjust
+
+        self.model_to_adjust_fpath = os.path.join(
+            self.subject.results_exp_path, self.model_to_adjust)
+
+        # Specify adjusted model
+        if not adjusted_model:
+            self.adjusted_model = '%s_adjusted.osim' % self.subject.name
+        else:
+            self.adjusted_model = adjusted_model
+
+        self.adjusted_model_fpath = os.path.join(trial.results_exp_path,
+            'rramodel', self.adjusted_model)
```

Comment:
I've actually moved this line out of SetupTask already, since it didn't support individual cycle task creation. But I'll keep that in mind for the future.